docs/general-development/prepare-to-set-up-and-configure-a-sharepoint-workflow-development-environment.md

```diff
@@ -64,7 +64,7 @@ If you have pre-release (that is, "Preview") versions of SharePoint Server, Work
 
 The release version of the Visual Studio workflow components and their related updates introduce important changes that enhance performance, scalability, and reliability. Unfortunately, these upgrades require you to update workflow projects that you created using the Preview tools.
 
-To make this process easier, we provide a conversion tool that you can get through CodePlex. The tool is called the  [SharePoint Workflow Converter for Visual Studio 2012](http://wfconverter.codeplex.com/).
+To make this process easier, we provide a conversion tool that you can get through CodePlex. The tool is called the SharePoint Workflow Converter for Visual Studio 2012.

```

Comment:
If the link is gone, remove the reference to the tool.

Reply:
The file is changed, for some reasons in not showing in UI.

Reply:
Changes applied as you suggested, for some reasons File Changes Diff UI is not reflecting changes as they were updated.